src/utils/formatURLForDisplay.ts

```diff
@@ -1,4 +1,8 @@
 export default function formatURLForDisplay(url: string) {
+  if (!url.includes('://')) {
+    const pretty = url.replace('www.', '');
+    return pretty.charAt(pretty.length - 1) === '/' ? pretty.substring(0, pretty.length - 1) : pretty;
+  }
```

Comment:
https://linear.app/rainbow/issue/APP-2451/expanded-state-crash-when-opening-ingested-token-that-includes-links

you do this bc u found a crash?